src/nls/ta/strings.js

```diff
@@ -0,0 +1,629 @@
+/*
+ * Copyright (c) 2012 Adobe Systems Incorporated. All rights reserved.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a
+ * copy of this software and associated documentation files (the "Software"),
+ * to deal in the Software without restriction, including without limitation
+ * the rights to use, copy, modify, merge, publish, distribute, sublicense,
+ * and/or sell copies of the Software, and to permit persons to whom the
+ * Software is furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+ * FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
+ * DEALINGS IN THE SOFTWARE.
+ *
+ */
+
+/*jslint vars: true, plusplus: true, devel: true, nomen: true, indent: 4, maxerr: 50 */
+/*global define */
+
+define({
+
+    /**
+     * Errors
+     */
+
+    // General file io error strings
+    "GENERIC_ERROR"                     : "(error {0})",
+    "NOT_FOUND_ERR"                     : "The file/directory could not be found.",
+    "NOT_READABLE_ERR"                  : "The file/directory could not be read.",
+    "EXCEEDS_MAX_FILE_SIZE"             : "Files larger than {0} MB cannot be opened in {APP_NAME}.",
```

Comment:
This would help other translators to contribute their translation, without
creating a new language.

Meanwhile I can make the complete translation and update. I just added
support for the language,  will contribute the enitre translation
progressively.

I felt this would, ease the other translators and just have to worry only
about the translation part.



On Thu, Mar 12, 2015 at 6:14 PM, Prashant Kumar Singh <
notifications@github.com> wrote:

> In src/nls/ta/strings.js
> <https://github.com/adobe/brackets/pull/10726#discussion_r26297979>:
>
> > + */
> > +
> > +/*jslint vars: true, plusplus: true, devel: true, nomen: true, indent: 4, maxerr: 50 */
> > +/*global define */
> > +
> > +define({
> > +
> > +    /**
> > +     * Errors
> > +     */
> > +
> > +    // General file io error strings
> > +    "GENERIC_ERROR"                     : "(error {0})",
> > +    "NOT_FOUND_ERR"                     : "The file/directory could not be found.",
> > +    "NOT_READABLE_ERR"                  : "The file/directory could not be read.",
> > +    "EXCEEDS_MAX_FILE_SIZE"             : "Files larger than {0} MB cannot be opened in {APP_NAME}.",
>
> Do you also want to add the translations for these strings? Without that
> they would just appear as English for Tamil
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/adobe/brackets/pull/10726/files#r26297979>.
>